model/versioning/Versioned.php

```diff
@@ -1196,12 +1196,96 @@ public function doPublish() {
 		$owner = $this->owner;
 		$owner->invokeWithExtensions('onBeforePublish');
 		$owner->write();
-		$owner->publish("Stage", "Live");
+		$owner->publish(static::DRAFT_STAGE, static::LIVE_STAGE);
 		$owner->invokeWithExtensions('onAfterPublish');
 		return true;
 	}
 
+	/**
+	 * Trigger publishing of owned objects
+	 */
+	public function onAfterPublish() {
+		$owner = $this->owner;
+
+		// Publish owned objects
+		$published = new ArrayList();
+		foreach ($owner->findOwned(false, $published) as $object) {
```

Comment:
Oh, this is going to be added to DataObject. Is that in this PR? I don't see it declared anywhere.